src/pages/studyView/table/CNAGenesTable.tsx

```diff
@@ -222,16 +222,30 @@ export class CNAGenesTable extends React.Component<ICNAGenesTablePros, {}> {
             headerRender: () => {
                 return <div style={{marginLeft: this.cellMargin[ColumnKey.FREQ]}}>Freq</div>
             },
-            render: (data: CopyNumberCountByGene) => <span
-                style={{
-                    flexDirection: 'row-reverse',
-                    display: 'flex',
-                    marginRight: this.cellMargin[ColumnKey.FREQ]
-                }}>{getFrequencyStr(data.frequency)}</span>,
-            sortBy: (data: CopyNumberCountByGene) => data.frequency,
+            render: (data: CopyNumberCountByGene) => {
+                const addTotalProfiledOverlay = () =>
+                    <span>{`# of samples profiled for CNA in this gene: ${data.numberOfSamplesProfiled}`}</span>;
+               return(
+                <div className={styles.displayFlex}>
```

Comment:
is there a need for this div and the flex box?  curious to me because it only has a single child.